sunpy/instr/rhessi.py

```diff
@@ -5,106 +5,32 @@
     .. warning:: This module is in development.
 
 """
-from __future__ import absolute_import, print_function
 
-import csv
-import posixpath
 import re
-import socket
-import warnings
+import csv
 from datetime import datetime, timedelta
 
 import numpy as np
-from dateutil.relativedelta import relativedelta
 
 from astropy import units as u
 
+import sunpy.io
 from sunpy.time import TimeRange, parse_time
 from sunpy.sun.sun import solar_semidiameter_angular_size
 from sunpy.coordinates import get_sunearth_distance
-import sunpy.map
-import sunpy.io
 
-from sunpy.extern.six.moves import urllib
-from sunpy.extern.six.moves.urllib.request import urlopen, urlretrieve
-from sunpy.extern.six.moves.urllib.error import URLError
-
-
-__all__ = ['get_obssumm_dbase_file', 'parse_obssumm_dbase_file',
```

Comment:
`get_obsumm_dbase_file`, `get_obssum_filename` and `get_obssumm_file` should all be marked as deprecated in 0.9 and say "use Fido".

Reply:
I can make a note and do a PR for that unless you really want to.

Reply:
@ehsteve Does this seem reasonable to you, or are there reasons to keep these as independent functions?

Reply:
I think we should keep that functionality here for two reasons with Fido calling it.
1] It is easier for someone to find the code for a particular instrument rather than it being buried in Fido.
2] I think there is more information in the dbase files that could be used for other reasons besides Fido.